src/base/lockfree_transaction_reclaimable.hpp

```diff
@@ -47,8 +47,12 @@ namespace lockfree
     class reclaimable_node
     {
       public:
-	reclaimable_node () = default;
-	virtual ~reclaimable_node () = 0;   // to force abstract class
+	reclaimable_node ()
```

Comment:
Is there other specialization than free_node for reclaimable_node ?
Do we need reclaimable_node abstract class ?
I am not sure the cost of accessing virtual table of reclaimable_node * instances from descriptor methods is to be ignored.

Reply:
Not sure, I will make some performance tests comparing this implementation with previous.